### ISSUE TITLE
MobileThin frozen graph

### ISSUE THREAD

**Issue by DeeperDeeper** [ildoonet/tf-pose-estimation#523]
Hi @ildoonet,
By investigating the code, I could not figure out the MobileNetThin configration (conv_width=?, conv_width2=?) graph  `/mobilenet_thin/graph_opt.pb` used for the inference !
For `models/graph/mobilenet_thin/graph_freeze.pb` it corresponds  to (conv_width=0.75, conv_width2=0.50) and I built the graph using tf.keras.layers and I matched the weights, however the results is not good!
For the `models/graph/mobilenet_thin/graph.pb`, I can't also guess the configuration. Because, you did not provide a checkpoint file in the pretrained folder, I can't also get the frozen graph from.
could you please give me the write configuration used for the inference. I want to build the same graph using `tf.keras.layers`. Thanks

**Comment by DeeperDeeper** [comment]
I have understand that graph_freeze.pb is the frozen graph and  graph_opt.pb is the optimized graph. By checking the weight of the first conv2d in both graph, I remark that are different. Is the optimization operation change the weights?
`graph_opt.pb:  'prefix/MobilenetV1/Conv2d_0/weights'`

> array([[[[-1.13355869e-03,  4.49307496e-03,  1.66561256e-06,
>           -9.12229385e-13,  9.25326999e-03,  2.47383351e-03,
>           -8.87035796e-08, -2.89668116e-12,  2.36324244e-03,
>            3.03913166e-12, -4.69298422e-04, -3.27827262e-12,
>           -9.02321015e-04,  1.64789586e-08,  4.20302531e-05,
>           -9.96988453e-03,  6.54694100e-04,  1.24835439e-12,...
> 

`graph_freeze.pb  'prefix/MobilenetV1/Conv2d_0/weights'`

> array([[[[-3.13527644e-01,  1.31075382e-01,  5.26713180e-08,
>           -2.88472283e-14,  4.17140037e-01,  6.28222227e-02,
>           -2.80505374e-09, -9.16011109e-14,  4.09043074e-01,
>            9.61057880e-14, -1.51180495e-02, -1.03668091e-13,
>           -3.16756010e-01,  5.21110488e-10,  6.19748281e-03,
>           -3.73009741e-01,  6.62895143e-02,  3.94764347e-14,
>           -6.93884240e-14,  3.25508714e-01, -2.02994362e-01,
>            1.98573574e-01,  1.44059643e-01,  6.87985855e-13],
>          [ 3.81347626e-01,  1.51933551e-01, -2.72923071e-08,....

any comment ?

**Comment by BraginIvan** [comment]
@DeeperDeeper It can happen because of fused batch norm, but I'm not sure
